src/pydata_sphinx_theme/assets/styles/extensions/_sphinx_design.scss

```diff
@@ -50,14 +50,14 @@ $sd-semantic-color-names: (
   * already define for our own needs.
   */
 $extra-semantic-colors: (
-  "dark": map-get($pst-semantic-colors, "text-base"),
```

Comment:
Note: not blocking - that this will change with the changes in #1174  as I encountered issues with using `*` while working on this